staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -176,6 +176,12 @@ type AuthenticationConfiguration struct {
 	// authenticators is neither defined nor stable across releases.  Since
 	// each JWT authenticator must have a unique issuer URL, at most one
 	// JWT authenticator will attempt to cryptographically validate the token.
+	//
+	// The minimum valid JWT payload must contain the following claims:
+	// - "iss" (issuer)
+	// - "sub" (subject)
```

Comment:
```suggestion
	// - "aud" (audience)
```

`sub` isn't required right?  Just any claim that is the for the username?

Reply:
yes, that's correct! should be fixed now